projects/ng-components/src/lib/controls/advanced-search/advanced-search.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="advanced-search-container">
+    <div class="advanced-search-filter">
+        <!--SEARCH FIELDS-->
+        <div class="advanced-search-filter-field-container">
+            <label class="advanced-search-filter-field-label"
+                   for="selectedField">
+                Πεδίο Αναζήτησης
+            </label>
+            <select [(ngModel)]="selectedField"
+                    (ngModelChange)="selectedFieldChanged()"
+                    name="selectedField"
+                    class="advanced-search-filter-field-select">
+                <ng-container *ngFor="let option of searchOptions">
+                    <option [ngValue]="option">{{ option.name }}</option>
+                </ng-container>
+            </select>
```

Comment:
Εδώ μπορείς να βάλεις το lib-drop-down που υπάρχει ήδη στη βιβλιοθήκη 
```suggestion
            <lib-drop-down-menu [options]="options" [selected]="selectedField" (selectedChanged)="selectedFieldChanged()"></lib-drop-down-menu>
```

Reply:
πρόσεξε το options πρέπει να είναι τύπου MenuOption[] δηλαδή μπορείς να κάνεις στο ngOnInit

`  public ngOnInit(): void { 
    this.searchOptions.forEach((option) => {
      this.options.push({
        text: option.name,
        value: option.field,
        data: option.dataType,
        description: option.placeholder,
        icon: undefined
      });
    });
  }
`